src/resources/stubs/header_metas.stub

```diff
@@ -0,0 +1,9 @@
+<meta name="apple-mobile-web-app-capable" content="yes" />
```

Comment:
I've just generated our icons again using https://realfavicongenerator.net which is my favorite, and it has a few stuff in addition to what is here. Namely the Windows stuff. Not sure that's really relevant any more in 2024... you should know better Pedro.

Here's the whole zip with it - I've chosed `/admin` as my prefix there, and `Backpack` as my app title:

[favicon_package_v0.16 (4).zip](https://github.com/user-attachments/files/16157735/favicon_package_v0.16.4.zip)